dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyAPI.kt

```diff
@@ -43,9 +43,10 @@ interface CompanyAPI {
         ResponseEntity<StoredCompany>
 
     @Operation(
-        summary = "Retrieve specific companies by name or just all companies from the data store.",
-        description = "Companies identified via the provided company name are retrieved. " +
-            "If company name is an empty string, all companies in the data store are returned."
+        summary = "Retrieve specific companies by name/identifier/index or just all companies from the data store.",
+        description = "Companies identified via the provided company name/identifier/index are retrieved. " +
+            "If an empty string is passed as search argument, all companies in the data store will be returned." +
```

Comment:
Müsste es nicht heißen "if **ONLY** and empty string is passed as search argument" ?

Weil wenn ich zwar einen empty string als search argument nehme, aber zusätzlich auf einen Index filtere, krieg ich ja nicht alle companies.

Reply:
geändert